aggregates/src/androidMain/kotlin/org/dhis2/mobile/aggregates/data/DataSetInstanceRepositoryImpl.kt

```diff
@@ -342,7 +342,7 @@ internal class DataSetInstanceRepositoryImpl(
                             ?: dataElementCategoryComboUid(it.uid),
                     )
                 },
-            ).mapIndexed { index, noGroupingDataSetElements ->
+            ).mapIndexedNotNull { index, noGroupingDataSetElements ->
```

Comment:
can we create a test to assert that an empty category option combo work as expected?

Reply:
I've been checking the best way to add a test and I think to include it in the configuration automated is the easiest. We need to ask Rohit, I think we can create a new ticket to implement it.